$OpenBSD$

Relax unicode-transforms bounds to the versions available in the OpenBSD
Haskell stack.

--- ../unicode-transforms-0.4.0.1/unicode-transforms.cabal
+++ ../unicode-transforms-0.4.0.1/unicode-transforms.cabal
@@ -82,14 +82,14 @@
   hs-source-dirs: .
   ghc-options: -Wall -fwarn-identities -fwarn-incomplete-record-updates -fwarn-incomplete-uni-patterns -fwarn-tabs
   build-depends:
-      base         >= 4.8 && < 4.17
+      base         >= 4.8 && < 4.21
     , unicode-data >= 0.2 && < 0.4
     , bytestring   >= 0.9 && < 0.12
-    , ghc-prim     >= 0.2 && < 0.9
+    , ghc-prim     >= 0.2 && < 0.13
 
     -- We depend on a lot of internal modules in text. We keep the upper bound
     -- inclusive of the latest stable version.
-    , text         >=1.1.1 && <=1.2.5.0 || >=2.0 && <2.1
+    , text         >=1.1.1 && <=1.2.5.0 || >=2.0 && <2.2
   if flag(dev)
     ghc-options: -O0
   else
